plugins/explore/src/components/ToolExplorerContent/ToolExplorerContent.tsx

```diff
@@ -13,27 +13,11 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
+import { Content, ContentHeader, SupportButton } from '@backstage/core';
 import React from 'react';
-import { makeStyles, Typography } from '@material-ui/core';
-import {
-  Content,
-  ContentHeader,
-  Header,
-  Page,
-  SupportButton,
-} from '@backstage/core';
-import ExploreCard, { CardData } from './ExploreCard';
-import { BackstageTheme } from '@backstage/theme';
+import { ToolCardGrid } from '../ToolCard';
 
-const useStyles = makeStyles<BackstageTheme>(theme => ({
-  container: {
-    display: 'grid',
-    gridTemplateColumns: 'repeat(auto-fill, 296px)',
-    gridGap: theme.spacing(3),
-    marginBottom: theme.spacing(6),
-  },
-}));
+// TODO: Provide API to configure the toolsCards
```

Comment:
If there is no way to access the available plugins, I would provide a static API that can be used to inject a custom list